Lib/test/test_descr.py

```diff
@@ -4757,6 +4762,22 @@ def __call__(self, arg):
         self.assertRegex(repr(method),
             r"<bound method qualname of <object object at .*>>")
 
+    @unittest.skipIf(_testcapi is None, 'need the _testcapi module')
```

Comment:
We can wrap the test with `@cpython_only`.

Reply:
I wanted to propose the same change, but IMHO @unittest.skipIf(_testcapi is None, ...) is fine since _testcapi is specific to CPython and it's a common pattern.

Reply:
`cpython_only` serves also documenting purpose. It is common to decorate all tests that use `_testcapi` with `cpython_only`.

Reply:
@jdemeyer: Would you mind to add @cpython_only? IMHO it's better to add it first, before "_testcapi is None".

Reply:
> Would you mind to add @cpython_only? IMHO it's better to add it first, before "_testcapi is None".

Do we really need both? Doesn't `@cpython_only` imply that `_testcapi` is available?